chaco/tools/simple_zoom.py

```diff
@@ -1,649 +0,0 @@
-""" Defines the SimpleZoom class.
```

Comment:
looks like this too was moved into `enable` as the viewport zoom tool and modified.

When we do the release, i think it will be good to highlight the fact that there are classes in enable which can be used by end users if they can't migrate to the chaco ZoomTool immediately.